x-pack/plugins/synthetics/public/apps/synthetics/components/settings/global_params/params_list.tsx

```diff
@@ -0,0 +1,273 @@
+      sortable: true,
+      render: (val: string) => <EuiText size="s">{val ?? '--'}</EuiText>,
+    },
+    {
+      name: i18n.translate('xpack.synthetics.settingsRoute.params.tags', {
+        defaultMessage: 'Tags',
+      }),
+      field: 'attributes.tags',
+      sortable: true,
+      render: (val: string[]) => {
+        const tags = val ?? [];
+        if (tags.length === 0) {
+          return <EuiText>--</EuiText>;
+        }
+        return (
+          <EuiFlexGroup gutterSize="xs" wrap>
+            {tags.map((tag) => (
+              <EuiFlexItem grow={false} key={tag}>
+                <EuiBadge>{tag}</EuiBadge>
+              </EuiFlexItem>
+            ))}
+          </EuiFlexGroup>
+        );
+      },
+    },
+    {
+      name: i18n.translate('xpack.synthetics.settingsRoute.params.actions', {
+        defaultMessage: 'Actions',
+      }),
+      actions: [
+        {
+          name: DELETE_PARAM,
+          description: DELETE_PARAM,
+          icon: 'trash',
+          color: 'danger',
+          type: 'icon',
+          onClick: (item: ListItem) => {
+            setDeleteParam([item]);
+            setIsDeleteModalVisible(true);
+          },
+          isPrimary: true,
+          'data-test-subj': 'action-delete',
+          enabled: () => canSave,
+        },
+        {
+          name: EDIT_PARAM,
+          description: EDIT_PARAM,
+          isPrimary: true,
+          icon: 'pencil',
+          type: 'icon',
+          onClick: (item: ListItem) => {
+            setIsEditingItem(item);
+          },
+          'data-test-subj': 'action-edit',
+          enabled: () => canSave,
+        },
+      ],
+    },
+  ];
+
+  const tagsList = items.reduce((acc, item) => {
+    const tags = item.attributes.tags || [];
+    return new Set([...acc, ...tags]);
+  }, new Set<string>());
+
+  const renderToolsLeft = () => {
+    if (selectedItems.length === 0) {
+      return;
+    }
+
+    return (
+      <EuiButton
+        color="danger"
+        onClick={() => {
+          setDeleteParam(selectedItems);
+          setIsDeleteModalVisible(true);
+        }}
+      >
+        Delete {selectedItems.length} params
+      </EuiButton>
+    );
+  };
+
+  const renderToolRight = () => {
+    return [
+      <AddParamFlyout
+        isEditingItem={isEditingItem}
+        setIsEditingItem={setIsEditingItem}
+        setRefreshList={setRefreshList}
+        items={items}
+      />,
+    ];
+  };
+
+  return (
+    <div>
+      <EuiText>
+        <FormattedMessage
+          id="xpack.synthetics.params.description"
+          // TODO: add link to docs
+          defaultMessage="Define variables and parameters that you can use in the configuration of browser and lightweight monitors, such as credentials or URLs. {learnMore}"
+          values={{
+            learnMore: (
+              <EuiLink
+                href="https://www.elastic.co/guide/en/observability/master/synthetics-params-secrets.html"
```

Comment:
Should use `docLinks.links.observability` instead of hardcoded values.

Reply:
it's not available there.